src/Shared/Components/SelectPicker/SelectPicker.component.tsx

```diff
@@ -373,52 +372,34 @@ const SelectPicker = <OptionValue, IsMulti extends boolean>({
-                                    NoOptionsMessage: renderNoOptionsMessage,
-                                }}
-                                closeMenuOnSelect={false}
-                                allowCreateWhileLoading={false}
-                                isValidNewOption={isValidNewOption}
-                                createOptionPosition="first"
-                                onCreateOption={handleCreateOption}
-                                renderMenuListFooter={!optionListError && renderMenuListFooter}
-                            />
-                        ) : (
-                            <ReactSelect
-                                {...props}
-                                {...commonProps}
-                                ref={selectRef as MutableRefObject<SelectInstance<SelectPickerOptionType<OptionValue>>>}
-                                components={{
-                                    IndicatorSeparator: null,
-                                    LoadingIndicator: SelectPickerLoadingIndicator,
-                                    DropdownIndicator: SelectPickerDropdownIndicator,
-                                    Control: renderControl,
-                                    Option: renderOption,
-                                    MenuList: SelectPickerMenuList,
-                                    ClearIndicator: SelectPickerClearIndicator,
-                                    ValueContainer: renderValueContainer,
-                                    NoOptionsMessage: renderNoOptionsMessage,
-                                }}
-                                renderMenuListFooter={!optionListError && renderMenuListFooter}
-                            />
-                        )}
+                        <CreatableSelect
+                            {...props}
+                            {...commonProps}
+                            isMulti={isMulti}
+                            ref={selectRef}
+                            components={{
+                                IndicatorSeparator: null,
+                                LoadingIndicator: SelectPickerLoadingIndicator,
+                                DropdownIndicator: SelectPickerDropdownIndicator,
+                                Control: renderControl,
+                                Option: renderOption,
+                                MenuList: SelectPickerMenuList,
+                                ClearIndicator: SelectPickerClearIndicator,
+                                ValueContainer: renderValueContainer,
+                                MultiValueLabel: renderMultiValueLabel,
+                                MultiValueRemove: SelectPickerMultiValueRemove,
+                                GroupHeading: renderGroupHeading,
+                                NoOptionsMessage: renderNoOptionsMessage,
+                            }}
+                            closeMenuOnSelect={closeMenuOnSelect}
```

Comment:
closeMenuOnSelect should be false for multi select.


```suggestion
                            closeMenuOnSelect={!isMulti || closeMenuOnSelect}
```